assets/javascripts/audit_log.js

```diff
@@ -4,6 +4,42 @@
 var audit_url;
 var ajax_url;
 
+function getURLForType(type, event_data) {
```

Comment:
This goes against our usual way of using the server-side `url_for` helper. I personally don't care much but just wanted to mention it.